From a43ccb08178951aae4aff524347956f51589d943 Mon Sep 17 00:00:00 2001
From: chandavath <rcrajagopal@habana.ai>
Date: Tue, 16 Jul 2024 12:35:55 +0300
Subject: [PATCH] [SW-179840] gaudi3: security.c is not used in erom pcie

security.c file is not used in gaudi3 erom pcie and which increases
erom pcie fw size. So moved this file inclusion to device
specific Makefile

Change-Id: I1ef5fafd29e423f8a9e0b87c0afe42a633441c14
Signed-off-by: chandavath <rcrajagopal@habana.ai>
---

diff --git a/app/ext_rom/Makefile b/app/ext_rom/Makefile
index d9aed2e..d00bce2 100644
--- a/app/ext_rom/Makefile
+++ b/app/ext_rom/Makefile
@@ -48,7 +48,6 @@
 	$(HL_DEV_ARCH)/../common/$(HL_TARGET).o \
 	$(HL_DEV_ARCH)/../common/common.o \
 	$(HL_DEV_ARCH)/common.o \
-	$(HL_DEV_ARCH)/../common/security.o \
 	erom_secondary.o
 
 EROM_RECOVERY_DEPS := \
diff --git a/arch/gaudi2/arc/Makefile b/arch/gaudi2/arc/Makefile
index 7650e53..51aafdc 100644
--- a/arch/gaudi2/arc/Makefile
+++ b/arch/gaudi2/arc/Makefile
@@ -7,4 +7,7 @@
 
 CFLAGS += -DPHY_FW_SIZE=$(PHY_FW_SIZE)
 
+EROM_PCIE_DEPS += \
+	$(HL_DEV_ARCH)/../common/security.o
+
 image_validation: image_validation_common
